### ISSUE TITLE
Admin Terminal Option

### ISSUE THREAD

**Issue by mpetuska** [kerol2r20/Windows-terminal-context-menu#28]
It'd be great if by right-clicking with SHIFT the menu would instead show options to open each terminal in elevated Administrator mode (prompting user for admin privileges if needed)

**Comment by brybalicious** [comment]
Yes, I do not understand which `settings.json` is being referred to in the Readme, where the RunAs flag can be set/invoked?
